scylla/src/policies/load_balancing/default.rs

```diff
@@ -1420,6 +1420,7 @@ mod tests {
                 None,
                 TabletsInfo::new(),
                 &HashMap::new(),
+                #[cfg(feature = "metrics")]
                 &Default::default(),
             )
```

Comment:
This, and few other changes likes this, are present in commit `metrics: unpub Metrics::new() `. Don't they belong to the commit that created the feature?